src/videojs-hls.js

```diff
@@ -676,8 +734,8 @@ videojs.Hls.prototype.findCurrentBuffered_ = function() {
   if (buffered && buffered.length) {
     // Search for a range containing the play-head
     for (i = 0; i < buffered.length; i++) {
-      if (buffered.start(i) <= currentTime &&
-          buffered.end(i) >= currentTime) {
+      if (buffered.start(i) <= (currentTime + 0.05) &&
+          buffered.end(i) >= (currentTime - 0.05)) {
```

Comment:
I'm not sure we ever came up with a good solution for that one yet. This also handles the case where our start isn't quite zero. This is an interim solution.